pkg/conversion/decode.go

```diff
@@ -19,6 +19,7 @@ package conversion
 import (
 	"errors"
 	"fmt"
+
```

Comment:
This is not necessary, but the change is consistent with other files, so I kept this change.